viite-UI/src/view/JunctionEditForm.js

```diff
@@ -0,0 +1,156 @@
+(function (root) {
+    root.JunctionEditForm = function (backend) {
+        var junctionIdForLink = 0;
+        var svgJunction =
```

Comment:
svg pictures are already defined. I think you can use them like this : 

`> <object type="image/svg+xml" data="images/**filename**.svg>`

I'm duplicating comments - we need to be carefull with merging this user-stories since they have a lot of duplicated code.

Reply:
Fixed in Viite-2034 branch, where JunctionEditForm.js is implemented